app/views/dashboard/player_dashboard.html.haml

```diff
@@ -8,57 +8,19 @@
       Dashboard
 %hr
 .row
-  .span8.no-display
+  .span8
     = render 'checkins/current_puzzle_info', locals: {puzzle: @puzzle, location_coordinates: @location_coordinates, next_location_coordinates: @next_location_coordinates}
 
     %hr
 
-    / #FIXME: Look up contact number from settings or Game Control team
     %h3
       Need help?
     %h4
       Contact Game Control:
-      %a{href: 'tel:+1-678-384-4464'} +1-678-384-4464
-    %h4 Tune in to Radio GAC
-    %ul
-      %li= link_to 'Listen in your own player', 'http://gchq.gac2014.com:8000/radio_gac'
```

Comment:
Perhaps we can move the radio code to a partial, rather than drop it entirely? I know it wasn't part of this game, but I'd hate to lose all that